File: FE/src/const/enum/category-type.ts
export enum CategoryType {
	MainType = 1,
	Genre = 2,
	WorldTheme = 3,
	MainCharacter = 4,
	Class = 5,
}

export const CategoryTypeLabel: Record<CategoryType, string> = {
  [CategoryType.MainType]: "Loại",
  [CategoryType.Genre]: "Thể loại",
  [CategoryType.WorldTheme]: "Bối cảnh thế giới",
  [CategoryType.MainCharacter]: "Nhân vật chính",
  [CategoryType.Class]: "Trường phái"
};

File: BE/Const/Enum/CategoryType.cs
using System.ComponentModel;

public enum CategoryType
{
    // [Description("Loại")] MainType = 1,
    [Description("Thể loại")] Genre = 1,
    [Description("Bối cảnh thế giới")] WorldTheme = 2,
    [Description("Nhân vật chính")] MainCharacter = 3,
    [Description("Trường phái")] Class = 4
}